cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
+#                     Michael Elsdörfer, Tony Hesjevik, Esben Haabendal,
+#                     Michael A. Smith
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
wrapdocker originally comes from https://github.com/jpetazzo/dind which is also apache2, but under different copyright. It's been heavily modified for our purposes tho. 

I've committed several times since it was added by @sttts. Does the commit hook only check modified files?